app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,30 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  const [lowerBoundError, setLowerBoundError] = useState('')
+  const [startingPointError, setStartingPointError] = useState('')
+  const [upperBoundError, setUpperBoundError] = useState('')
+
+  useEffect(() => {
+    Number(lowerBound) < 0
+      ? setLowerBoundError('Value cannot be negative')
+      : upperBound && Number(upperBound) !== 0 && Number(lowerBound) > Number(upperBound)
```

Comment:
It works to convert to `Number`s then do the arithmetic, but in general we want to avoid type conversions wherever practicable. So for example, on this line we should do as follows:
`upperBound && !upperBound.eq(0) && lowerBound.gt(upperBound)`.

It's super nitpicky, and not completely necessary but it's best practice and super useful to get used to using the `BigNumber` type. 

For reference: https://docs.ethers.io/v4/api-utils.html#big-numbers

Reply:
i see i see. i'll do this from now on, thanks.